project.yaml

```diff
@@ -992,7 +1011,7 @@ Layer:
                 foot,
                 bicycle,
                 tracktype,
-                'null',
+                'null' AS surface, -- Should be a SQL NULL?
```

Comment:
null is here as it makes little sense to query surface for railways.

on the other hand querying some other tags like tracktype, bicycle, foot for railways is also rather pointless so there is no good reason to do the same for surface.

Also, for aeroway querying surface is not completely pointless.

Overall replacing this null by querying surface may be a good idea.